justfile

```diff
@@ -0,0 +1,57 @@
+set dotenv-load := true
+
+start: build
+  npm start
+
+start-qt:
+  PATH="${PATH}:$(pwd)/bin" bin/matbas
+
+build: 
+  make MATBAS_BUILD="${MATBAS_BUILD}"
+
+release:
+  make MATBAS_BUILD=release
+
+dist:
+  make dist MATBAS_BUILD="${MATBAS_BUILD}"
+
+check: dist
+  npm run check
+
+format:
+  npm run format
+  clang-format -i core/*.cpp core/*.h
+
+format-markdown file:
```

Comment:
I wonder if this is still necessary? I believe it mostly exists to knock out newlines. It may also be solveable with `pandoc`.